### PR TITLE
[Github]Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ Feel free to ping committers for the review!
 ## Check list
 
 * [ ] Code changed are covered with tests, or it does not need tests for reason:
-* [ ] Change does not need document change, or I will submit document change to https://github.com/apache/incubator-seatunnel-website later
+* [ ] Update the documentation to include this new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
```
Comment (leo65535) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: `include` to `describe`

**Review by leo65535 — APPROVED** [review]
Overall LGTM, left one small comment.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (CalvinKirs) [reply .github/PULL_REQUEST_TEMPLATE.md]: well,thx:)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -23,4 +23,4 @@ Feel free to ping committers for the review!
 ## Check list
 
 * [ ] Code changed are covered with tests, or it does not need tests for reason:
-* [ ] Change does not need document change, or I will submit document change to https://github.com/apache/incubator-seatunnel-website later
+* [ ] Update the documentation to describe this new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
```
Comment (zhongjiajie) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: I prefer to use keyword `if this pull request add new feature` or something like that

Comment (leo65535) [reply .github/PULL_REQUEST_TEMPLATE.md:26]: how about `If necessary, please update the documentation to describe the new features.`

Comment (CalvinKirs) [reply .github/PULL_REQUEST_TEMPLATE.md]: thx,done~

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**